nucleus/glassfish-jul-extension/src/main/java/org/glassfish/main/jul/cfg/GlassFishLoggingConstants.java

```diff
@@ -67,6 +68,23 @@ public class GlassFishLoggingConstants {
     /** If this key is set to true, GJULE will print really detailed tracing info to the standard output */
     public static final String KEY_TRACING_ENABLED = "org.glassfish.main.jul.tracing.enabled";
 
+    /**
+     * If this key is set to true, GJULE will detect the caller class and method from stacktrace,
+     * which is quite expensive operation affecting logging throughput.
+     * <p>
+     * If it is set to null, GJULE will not perform such detection.
```

Comment:
I don't understand that. It says: _if set to **null**_ but the code seems to check for `false`.

I'm not very familiar with properties handling here, but _set to null_ is different case than _not set_, isn't it?

Just would like to know what are my options here :grin: and where the `false` from 
```java
        if (Boolean.FALSE.toString().equalsIgnoreCase(value)) {
            return false;
        }
```
fits in.

Reply:
You are right, I will take a sleep and I will read and fix it after myself :-)
Thanks for noticing that!
And I should add that to the documentation too.